src/main/java/com/linagora/james/mailets/GuessClassificationMailet.java

```diff
@@ -97,6 +111,12 @@ public GuessClassificationMailet() {
     @Override
     public void init() throws MessagingException {
         LOGGER.debug("init GuessClassificationMailet");
+        executorService = Executors.newFixedThreadPool(
+            MailetUtil.getInitParameterAsStrictlyPositiveInteger(
+                getInitParameter(THREAD_COUNT),
+                THREAD_COUNT_DEFAULT_VALUE));
```

Comment:
Is it adding a value to have a different thread count than 2? Else we can simply remove this parameter, I'm not sure it is necessary to expose it.